src/app/products/manage-tax-components/edit-tax-component/edit-tax-component.component.html

```diff
@@ -0,0 +1,72 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="taxComponentForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <mat-form-field>
+            <mat-label>Name</mat-label>
+            <input matInput required formControlName="name">
+            <mat-error *ngIf="taxComponentForm.controls.name.hasError('required')">
+              Name is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Percentage</mat-label>
+            <input matInput required formControlName="percentage">
+            <mat-error *ngIf="taxComponentForm.controls.name.hasError('required')">
+              Percentage is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>		
+
+          <div fxLayout="row wrap" class="content">
+
+	          <div fxFlex="50%" class="mat-body-strong" *ngIf="taxComponentData.creditAccountType">
+	          	Credit Account Type
+	          </div>
+
+	          <div fxFlex="50%" *ngIf="taxComponentData.creditAccountType">
+	            {{ taxComponentData.creditAccountType.value }}
+	          </div>
+
+	          <br>
+	          
+	          <div fxFlex="50%" class="mat-body-strong" *ngIf="taxComponentData.creditAccount">
+	            Credit Account
+	          </div>
+
+	          <div fxFlex="50%" *ngIf="taxComponentData.creditAccount">
+	            {{ taxComponentData.creditAccount.name }}
+	          </div>
+
+	      </div>
```

Comment:
@spider0061 can you replace this with disabled form fields please then we will merge.